src/cueUtilities.js

```diff
@@ -90,12 +91,37 @@ module.exports = function (params, cy, api) {
         ctx.clearRect(0, 0, w, h);
       }
 
+      function selectedGroupsContains(group) {
+        return selectedGroups.find(node => node.id() === group.id()) !== undefined;
+      }
+
+      function anyNodeHasACue() {
+        return hoveredGroup || selectedGroups.length > 0;
+      }
+
+      function hoveredGroupExistsAndNotInSelectedGroup() {
+        return hoveredGroup && !selectedGroupsContains(hoveredGroup);
+      }
+
+      function drawCuesForSelectedGroups() {
+        selectedGroups.forEach(group => drawExpandCollapseCue(group))
+      }
+
+      function removeGroupFromSelectedGroups(node) {
+        selectedGroups = selectedGroups.filter(group => group.id() !== node.id());
+      }
+
+      function mouseIsHoveringOver(node) {
```

Comment:
Our pattern for boolean functions is to start the name with `is`, so this should be `isMouseHoveringOver`. Revisit the other boolean functions above as well. (`does` or `are` might also be acceptable starting words if `is` sounds awkward.)

Reply:
As mentioned in my second sentence, your other boolean functions (`selectedGroupsContains`, 'anyNodeHasACue`, `hoveredGroupExistsAndNotInSelectedGroup`) should also start with `is` (or `are` or `does`) to give the indication that it will return a boolean.